Editor/GitLogWindow.cs

```diff
@@ -106,14 +176,13 @@ struct LogGraphCell
         bool HideLog => !string.IsNullOrEmpty(LockedHash);
         
         float FilesPanelHeight => HideLog ? position.height : verticalSplitterState.RealSizes[1];
-        float InfoPanelWidth => HideLog ? 0 : DefaultInfoPanelWidth;
+        float InfoPanelWidth => HideLog ? 0 : Screen.width / 2;
```

Comment:
I think, some predefined size is better, because you can set up a better layout for your fancy info window :)

Reply:
If there was a setting or tweaks system somewhere... Or if would be possible to modify line at center